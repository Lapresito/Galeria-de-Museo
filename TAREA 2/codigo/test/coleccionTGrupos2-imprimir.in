crearColeccionGrupos
imprimirColeccionGrupos
imprimirInvertidaColeccionGrupos
crearGrupoABB
crearVisitante 4 Zafiro Garcia 22
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 8 Filemon Ruiz 52
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 5 Brunilda Fernandez 22
agregarAGrupoABB
agregarAColeccionGrupos
imprimirColeccionGrupos
imprimirInvertidaColeccionGrupos
crearGrupoABB
crearVisitante 6 Brunilda Martinez 24
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 5 Artemio Garcia 22
agregarAGrupoABB
agregarAColeccionGrupos
imprimirColeccionGrupos
imprimirInvertidaColeccionGrupos
liberarColeccionGrupos
Fin
