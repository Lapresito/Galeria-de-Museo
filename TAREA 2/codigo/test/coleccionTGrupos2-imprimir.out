Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> La coleccion de grupos ha sido creado de forma exitosa.
2> Coleccion de grupos:
3> Coleccion de grupos:
4> El grupoABB ha sido creado de forma exitosa.
5> Visitante creadox de forma exitosa.
6> Se ha agregado el visitante al grupo de forma exitosa.
7> Se ha agregado el grupo a la coleccion de forma exitosa.
8> El grupoABB ha sido creado de forma exitosa.
9> Visitante creadox de forma exitosa.
10> Se ha agregado el visitante al grupo de forma exitosa.
11> Se ha agregado el grupo a la coleccion de forma exitosa.
12> El grupoABB ha sido creado de forma exitosa.
13> Visitante creadox de forma exitosa.
14> Se ha agregado el visitante al grupo de forma exitosa.
15> Se ha agregado el grupo a la coleccion de forma exitosa.
16> Coleccion de grupos:
Grupo con edad promedio 22.00:
Visitante Zafiro Garcia
Id: 4
Edad: 22
Grupo con edad promedio 22.00:
Visitante Brunilda Fernandez
Id: 5
Edad: 22
Grupo con edad promedio 52.00:
Visitante Filemon Ruiz
Id: 8
Edad: 52
17> Coleccion de grupos:
Grupo con edad promedio 52.00:
Visitante Filemon Ruiz
Id: 8
Edad: 52
Grupo con edad promedio 22.00:
Visitante Brunilda Fernandez
Id: 5
Edad: 22
Grupo con edad promedio 22.00:
Visitante Zafiro Garcia
Id: 4
Edad: 22
18> El grupoABB ha sido creado de forma exitosa.
19> Visitante creadox de forma exitosa.
20> Se ha agregado el visitante al grupo de forma exitosa.
21> Se ha agregado el grupo a la coleccion de forma exitosa.
22> El grupoABB ha sido creado de forma exitosa.
23> Visitante creadox de forma exitosa.
24> Se ha agregado el visitante al grupo de forma exitosa.
25> Se ha agregado el grupo a la coleccion de forma exitosa.
26> Coleccion de grupos:
Grupo con edad promedio 22.00:
Visitante Zafiro Garcia
Id: 4
Edad: 22
Grupo con edad promedio 22.00:
Visitante Brunilda Fernandez
Id: 5
Edad: 22
Grupo con edad promedio 22.00:
Visitante Artemio Garcia
Id: 5
Edad: 22
Grupo con edad promedio 24.00:
Visitante Brunilda Martinez
Id: 6
Edad: 24
Grupo con edad promedio 52.00:
Visitante Filemon Ruiz
Id: 8
Edad: 52
27> Coleccion de grupos:
Grupo con edad promedio 52.00:
Visitante Filemon Ruiz
Id: 8
Edad: 52
Grupo con edad promedio 24.00:
Visitante Brunilda Martinez
Id: 6
Edad: 24
Grupo con edad promedio 22.00:
Visitante Artemio Garcia
Id: 5
Edad: 22
Grupo con edad promedio 22.00:
Visitante Brunilda Fernandez
Id: 5
Edad: 22
Grupo con edad promedio 22.00:
Visitante Zafiro Garcia
Id: 4
Edad: 22
28> Coleccion de grupos liberada
29> Fin.
